coral/src/app/features/topics/schema-request/TopicSchemaRequest.test.tsx

```diff
@@ -216,11 +248,16 @@ describe("TopicSchemaRequest", () => {
         ...mockedGetSchemaRegistryEnvironments,
       ]);
 
-      customRender(<TopicSchemaRequest topicName={testTopicName} />, {
-        queryClient: true,
-        memoryRouter: true,
-        customRoutePath: "/topic/testtopic/request-schema?env=HELLO",
-      });
+      customRender(
+        <AquariumContext>
```

Comment:
Seeing this everywhere, I wonder if we could have it as an option in `customRender` ? `withAquariumContext: boolean`